nupic/frameworks/opf/clamodel.py

```diff
@@ -419,7 +423,7 @@ def run(self, inputRecord):
     if self.__logger.isEnabledFor(logging.DEBUG):
       self.__logger.debug("inputRecord: %r, results: %r" % (inputRecord,
                                                             results))
-
+    results.iterationTime = time.clock() - startTime
```

Comment:
I'm not familiar with the code you're referring to, so I'll defer to @scottpurdy on this one.